docs/consensus/pot.md

```diff
@@ -0,0 +1,58 @@
+Suppose in the first year of the blockchain's operation, all farmers were honest and have collectively pledged 100TB of storage. Now, suppose by the second year the total storage pledged reached 1PB, out of which an adversary has dedicated 200TB worth of storage. At no point does an adversary control more than 20% of storage, which is way less than a majority. However, using his 200TB, an adversary could rewrite the past year's history by participating in all past lotteries to win blocks back to the genesis and then grow a chain instantaneously from the genesis to surpass the current longest chain. This is possible because the adversary's resources are enough to win a disproportionately large number of past lotteries compared to his share of total storage. Such a long-range rewrite seriously threatens the security and immutability of blockchain history.
+On the other hand, in Bitcoin it takes a long time to mine such a chain from the past and that chain will always be behind the current longest chain. Thus, PoW enforces an arrow of time, meaning nodes cannot “go back in time” to mine blocks. This property is key to tolerating a fully dynamic honest and adversarial participation.
+
+## Arrow of Time
+
+Subspace's Proof-of-Time component addresses long-range attacks by enforcing an arrow of time similar to PoW protocols. PoT guarantees a certain amount of wall-clock time must elapse between block proposals, preventing an adversary from rewriting history by "going back in time". Similar to PoW, Proof-of-Time is constrained physically, however it is not parallelizable (technically, it is proof of _sequential_ work). We prevent the aforementioned attack by integrating the blockchain with a Proof-of-Time process. The attacker can not immediately generate a years-long fork on the spot even with faster hardware.
+The elapsed time guarantee is achieved by iterative evaluation of an inherently sequential function. The output of such a function is unpredictable and is used to build a randomness beacon for block challenges. 
+
+## Timekeeping
+
+For the task of running the time-chain Subspace introduces a new role for nodes called Timekeepers. Timekeepers are responsible for evaluating the delay function and announcing the outputs to other nodes. Anyone can become a Timekeeper as long as they have a sufficiently powerful CPU to be able to evaluate the delay function within the target time slot duration of 1 second. A timekeeper can also be a farmer and participate in block production or an operator executing computation on a domain. 
+A single timekeeper is sufficient for the security of the protocol, but for robustness and decentralization there will be multiple timekeepers running in parallel. We invite interested parties to run timekeeper component on their nodes to ensure the security and decentralization of the protocol. Domain operators may be more suited for the task since they likely already have powerful hardware.
```

Comment:
>A single timekeeper is sufficient for the security of the protocol, 

Does it need to be honest?

>but for robustness and decentralization there will be multiple timekeepers running in parallel.

I'm not sure that I get this, how can we guarantee that there will be multiple ones?

Reply:
Right, it needs to be honest. In worst case, we will run multiple in-house but we also have a few eager community members. So we can't "guarantee" I guess but it will likely be the case

Reply:
If subspace runs several timekeepers, from security perspective it is really as if only one is being run -- it definitely don't achieve decentralisation. I just think we should soften the language here.

Reply:
"there will be" --> "there should be"